graphblas/operator.py

```diff
@@ -2475,6 +2546,18 @@ def get_semiring(monoid, binaryop, name=None):
     "-": unary.ainv,
     "~": unary.lnot,
 }
+_str_to_select = {
+    "<": select.valuelt,
+    ">": select.valuegt,
+    "<=": select.valuele,
+    ">=": select.valuege,
+    "!=": select.valuene,
+    "==": select.valueeq,
+    "col<=": select.colle,
+    "col>": select.colgt,
+    "row<=": select.rowle,
+    "row>": select.rowgt,
+}
```

Comment:
Can we also add `"col<"`, `"row<"`, `"col>="`, and `"row>="` with recipes to make them work without too much hassle?

Reply:
Let's hold off on this for now. There is no easy way to make this happen. Hopefully with user-defined IndexUnaryOps, this will have an easier solution.